install.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 # Check if the script is running as root
-[ "$EUID" -eq 0 ] || exec sudo bash "$0" "$@"
+# Initialize the elevated variable to ensure we can run commands as root
+if [ "$EUID" -eq 0 ]; then
+    elevated="bash -c"  # No need for sudo or su if running as root
```

Comment:
`elevated=""`

Setting it to the empty string means it just goes away - setting it to bash means a whole new shell.